source/elements/oneDPL/source/parallel_api/parallel_range_api.rst

```diff
@@ -868,7 +868,8 @@ In-place Mutating Operations
     // replace
     template <typename ExecutionPolicy, std::ranges::random_access_range R,
               typename Proj = std::identity,
-              typename T1 = /*projected-value-type*/<std::ranges::iterator_t<R>, Proj>, typename T2 = T1>
+              typename T1 = /*projected-value-type*/<std::ranges::iterator_t<R>, Proj>,
+              typename T2 = std::ranges::range_value_t<R>>
```

Comment:
This asymmetry between `old_value` and `new_value` seems odd at first, given their names.

However, I understand that this asymmetry is necessary. It's like replacing by key. We specify a key (projection applied), `old_value`, and we specify a key-value pair, `new_value`, to replace a found entry with.

I agree that we should act proactively. But, I would anticipate more changes in the standard, e.g. different names of the arguments, which is not crucial.